### DISCUSSION TITLE
what should be used for the plan_id for an entity that doesn't have an EIN?

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#687] [Q&A]
There are cases where a plan does not have a HIOS ID and the entity for which the plan was created (the plan sponsor) does not have an EIN.  What should be used as the plan_id in these cases?

An example is a plan that is for a state or local government entity.

Currently it seems that what is being often used is the EIN of the insurance company that is acting as the TPA.

**Comment by karth295** [comment]
Government entities do have a federal taxpayer identification number (so a TIN that isn't an EIN):
* Docs from the IRS: https://www.irs.gov/government-entities/federal-state-local-governments/governmental-information-letter#:~:text=A%20government%20entity%20may%20use,subject%20to%20federal%20income%20tax.
* Here's the state of Iowa's number, for example: https://tax.fo.uiowa.edu/sites/tax.fo.uiowa.edu/files/IRS%20Form%204076C.pdf
* California also mentions that identifier: https://fiscal.ca.gov/user-support/cal-eprocure-resources/local-governments/

Maybe it's worth opening a PR to expand the schema to create a new plan_id_type "TIN" as a separate category from "EIN"?

**Reply by nmayle** [reply]
Thank you.  I didn't realize that they did exist.  Does the space of these overlap with the space of EINs?  Do we need a separate type?

Would that be used as the plan_id if the plan is a self-insured plan for a state's employees or a city's?

Right now it is hard to locate these plans.
